gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/test/java/io/gravitee/rest/api/portal/rest/resource/UserResourceTest.java

```diff
@@ -231,4 +238,14 @@ public void shouldGetNoContent() throws IOException {
         final Response response = target().path("avatar").request().get();
         assertEquals(OK_200, response.getStatus());
     }
+
+    private UserEntity mockUserEntity(String email) {
+        UserEntity userEntity = mock(UserEntity.class);
+        if(email != null) {
+            when(userEntity.getEmail()).thenReturn(email);
+        }
+        when(userEntity.getSource()).thenReturn("gravitee");
+        return userEntity;
```

Comment:
`UserEntity` is a POJO don’t need to mock it


```suggestion
        UserEntity entity = UserEntity.builder().source("gravitee").build();
        if (email != null) {
            entity.setEmail(email);
        }
        return userEntity;
```

Reply:
Ok, et au début j'avais créer une méthode en pensant avoir à le réutiliser dans les autres tests mais non